sqlite/migrations/0002_create_annotations_tables.sql

```diff
@@ -0,0 +1,36 @@
+-- The user_version should match the "000X" from the file name
+-- Ex: 0001_create_notebooks_table should have a user_verison of 1
+PRAGMA user_version=2;
+
+-- Create the initial table to store streams
+CREATE TABLE streams (
+  id VARCHAR(16) UNIQUE NOT NULL,
+  org_id VARCHAR(16) NOT NULL,
+  name TEXT NOT NULL,
+  description TEXT NOT NULL,
+  created_at TIMESTAMP NOT NULL,
+  updated_at TIMESTAMP NOT NULL,
+
+  PRIMARY KEY ("org_id", "name")
```

Comment:
I'm surprised `id` isn't a part of this.

I think we should instead have the PK be ID + org ID like below, with a uniqueness constraint + index on `name`. I'm worried having `name` in the PK will cause headaches when users try to update stream names.

Reply:
Hrm, we can't have `name` be unique since data from multiple orgs will be in this table and each org can have a stream with the same name. The constraint is really that a stream name must be unique within an org, so it's the combination of `org_id` + `name` that enforces that constraint.

I do think that the `id` column should have an index so I added that.

If you think it'd be more clear, we could do the ID + org ID PK combo, and then add a unique constraint on org ID + name?

Reply:
Discussed on Slack, I think PK=ID and a uniqueness constraint on org_id+name would be the best move.